.ci/schedule-tests/src/combine-statistics.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs')
+const path = require('path')
+const AdmZip = require('adm-zip')
+
+const processTextFile = (fileContent, durations) => {
+  const lines = fileContent.split(/\n/)
+
+  for (const line of lines) {
+    const [suite, testFile, duration] = line.split(/\s+/)
```

Comment:
Splitting on a constant string would be faster indeed, but it does not really matter at this point.  There is no intrinsic caching, so the regular expression is compiled in each iteration.  Amazingly, the speed increase in my microbenchmark was only 40%.

I'll move to JSON with this, so it will be much slower anyway :)